### DISCUSSION TITLE
Support an MS Windows-like ClickTo focus behaviour

### DISCUSSION THREAD

**Discussion by heisencoder** [leftwm/leftwm#929] [Ideas]
Would it be possible to have a `focus_behaviour` that emulates as close as possible the behavior of MS Windows?

The closest that I've found so far is setting `focus_behaviour` to `ClickTo`.  This behavior matches that of MacOS instead of Windows in that a mouse click will select the new window, but will not (necessary) then forward that mouse-click to the window itself, requiring a second click to modify the window.

Note, however, that this behavior is not consistent across all applications.  Chrome (mostly) acts immediately on the new click, but gnome-terminal does not register the click.  As such, I'm not quite sure if this is a bug report discussion or a feature request.

In any case, I'm hoping that the MS Windows behavior could be supported, possibly via a new `focus_behaviour` enum.

Thanks!

**Comment by VuiMuich** [comment]
Let's call this an enhancement 😄

I suppose this would be a relatively easy change.

Any suggestion for the name of this mode?
`ClickToAndForward`?

**Reply by heisencoder** [reply]
Sounds good to me! :)

**Comment by hertg** [comment]
I recently opened an issue for this exact request #1013
